restlicsharpclient/restliclientintegrationtest/DataTemplateTests.cs

```diff
@@ -16,15 +16,28 @@
 
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
+using com.linkedin.restli.test.api;
+
 namespace restlicsharpclient.restliclientintegrationtest
 {
     [TestClass]
-    public class DummyClientIntegrationTests
+    public class DataTemplateTests
     {
         [TestMethod]
-        public void DummyClientIntegration_Pass()
+        public void DataTemplate_BuildGreeting()
```

Comment:
No need to prefix with DataTemplate since this test is in DataTemplateTests class.